README.md

```diff
@@ -151,117 +156,154 @@ controls from scratch. Custom security controls implementation and not using
 the functionality provided by modern frameworks/well-known libraries will
 commonly be the root cause for many security issues you will work with.**
 
-1.  Main programming language: **Java**
+#### Main programming language: **Java**
 
-Widely used by the enterprise, it's not going away anytime soon. Unlike **.Net /
-C#**, it's free to use and learn. Also, it's not restricted to the Microsoft
-stack. Combined with a widespread community of helpful and friendly developers,
-Java is a great first language to learn.
+Widely used by the enterprise, it's not going away anytime soon. Unlike
+proprietary technology stacks like Microsoft's (.Net/C#) or Apple's, it's free
+to use and learn. Combined with a widespread community of helpful and friendly
+developers, Java is a great first language to learn.
 
-If you don't like Java, our other recommendation would be **Go**, but
-it's not currently very popular in the world of enterprise.
+If you don't like Java, our other recommendation would be **Go**, but it's not
```

Comment:
A brief explanation of why Go is recommended would be helpful here

Reply:
Added in the next commit. Just the same quality of 'why' as for Java: popularity and feedback we heard. What I won't mention there: basically, there are not so many popular languages to recommend: C, C++ -- filtered out, not popular. Java and C#/VB .NET - mentioned already,
Pascal - joke, Kotlin / Groovy - built on top of JVM, so in this text scope its better to recommend starting with Java, Go and Rust remain =) But rust is far less seen in Enterprise. Hence. Go :)